kalite/distributed/static/js/distributed/user/views.js

```diff
@@ -319,7 +320,7 @@ var LoginView = BaseView.extend({
         // has finished loading.
         this.set_login_button_state();
         this.$(".password-btn").removeAttr("disabled");
-        this.$("#id_username").on( "autocompletesearch", this.set_login_button_state);
+        this.$("#id_username").on(this.set_login_button_state);
```

Comment:
That looks weird? The `on` method is missing an event label, so it will use the `set_login_button_state` as event label, which will just cause no events to be caught. It's equivalent to removing the whole line?

http://api.jquery.com/on/